src/events/components/DetailView/PublicAttendees/index.tsx

```diff
@@ -1,29 +1,46 @@
-import React, { FC, useEffect } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 
 import { useDispatch, useSelector } from 'core/redux/hooks';
 import { selectPublicAttendeesForEventId } from 'events/selectors/publicAttendee';
 import { fetchPublicAttendeesByEventId } from 'events/slices/publicAttendees';
 
 import { Attendee } from './Attendee';
 import style from './PublicAttendees.less';
+import { IPublicAttendee } from 'events/models/Attendee';
+import { Button, Modal } from '@dotkomonline/design-system';
 
 interface IProps {
   eventId: number;
+  isAttending: boolean;
```

Comment:
If I recall correctly the rules for showing attendees should be that you are either eligible for signup or attending the event.

Looks like the API doesn't actually restrict it, which it should. https://github.com/dotkom/onlineweb4/blob/develop/apps/events/api/views.py#L125

Current view logic in OW4 can be found her: https://github.com/dotkom/onlineweb4/blob/develop/templates/events/details.html#L400
(where `user_status.status` is a boolean that refers to if the user is eligible for signup I think?)

Reply:
That should be a quite easy fix!

Reply:
Has been fixed by adding the `canAttend` prop and check if null (not logged in) and it's status (true if the user eligible).